com.geekbench.Geekbench6.yml

```diff
@@ -0,0 +1,31 @@
+app-id: com.geekbench.Geekbench6
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: geekbench6
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
```

Comment:
does it really support wayland?

Reply:
not sure, the benchmark itself never shows on screen. I figured giving this permission would be fine due to https://docs.flathub.org/docs/for-app-authors/requirements#unrestricted-permissions but we can just remove it.

Reply:
hence the question

Reply:
Removed it and didn't seem to change the vulkan score (might have even improved it slightly, but that might be moot)

Reply:
Maybe not as moot as I want too, probably need to run before after again, without working on the PC when it runs

https://browser.geekbench.com/v6/compute/compare/677002?baseline=678823

Reply:
fallback-x11 alone is not valid. It should be `x11` if it doesn't support wayland.

Reply:
Should be changed